src/services/completions.ts

```diff
@@ -360,6 +369,18 @@ namespace ts.Completions {
         return undefined;
     }
 
+    function getStringLiteralCompletionsFromIndexedAccess(node: IndexedAccessTypeNode, typeChecker: TypeChecker, target: ScriptTarget, log: Log): CompletionInfo | undefined {

```

Comment:
Since this is mostly the same as `getStringLiteralCompletionEntriesFromElementAccess`, I would combine them into `getStringLiteralCompletionEntriesFromElementAccessOrIndexedAccess`, and call them with the appropriate `type` and `stringLiteralNode`.

Reply:
That definitely makes sense. Would you be okay with `stringLiteralNode: Node` as a parameter, or would you prefer `stringLiteralNode: StringLiteral | NoSubstitutionTemplateLiteral`?